app/assets/javascripts/directives/url_validation.js

```diff
@@ -11,14 +11,15 @@ ManageIQ.angular.app.directive('urlValidation', ['nodeValidator', function(nodeV
       var options = {
         protocols: ['http', 'https', 'ssh'],
         require_tld: true,
-        require_protocol: true,
```

Comment:
any reason for this bit? (now that we're matching file separately)

Reply:
~~Mentioned it in the commit message, but it is basically unnecessary.  It uses this line:~~

I was going to say the above, but then when I was about to reference it to explain why, I realized I was wrong:

https://github.com/gkaimakas/angular.validators/blob/cc0204498bb57333082b0a70f2fd71beee86db4f/angular.validators.js#L329-L332

Basically, at first glance it looked like only one of the two were necessary, and if you had `require_valid_protocol`, you wouldn't need this one.  But looking at it again it seems if there is no protocol and `:require_protocol` is false (the default), it will silently allow that even if you are requiring specific protocols...

Anyway, I will put this line back.  Thanks!